cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/templates/etl/batch/sinks/StructuredRecordToCubeFact.java

```diff
@@ -14,7 +14,7 @@
  * the License.
  */
 
-package co.cask.cdap.templates.etl.transforms;
+package co.cask.cdap.templates.etl.batch.sinks;
```

Comment:
can I move it along with https://issues.cask.co/browse/CDAP-2221 (Implement realtime CubeSink)?